<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Norwester</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #05B893;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
 border-width: 10px;
 border-radius: 20px;
border: 2px solid #707070;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_pin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #5A6463</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Norwester</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #5A6463;
font-size: 18px;
border: none;
</string>
     </property>
     <property name="text">
      <string>PIN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Norwester</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#007871;
font-size: 15px;
border: none;
</string>
     </property>
     <property name="text">
      <string>BY LOGGING IN YOU ARE AGREEING TO 
 USER TERMS AND SERVICES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signIn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Norwester</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #05B893;
color: white;
font-size: 15px;
 border-width: 5px;
border: none;
 border-radius:7px;</string>
     </property>
     <property name="text">
      <string>SIGN IN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_accN">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>61</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #5A6463</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Norwester</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #5A6463;
font-size: 18px;
border: none;

</string>
     </property>
     <property name="text">
      <string>ACCOUNT NUMBER</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_page_warning">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Hebrew</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: red;
font-size: 15px;
border: none;
border-bottom: 1px red solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_mainLogo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Norwester</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font-size: 28px;
color: white;</string>
    </property>
    <property name="text">
     <string>THE UNITED ATM OF
WASSEYPUR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_accN</tabstop>
  <tabstop>lineEdit_pin</tabstop>
  <tabstop>pushButton_signIn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
